components/AlertNotification/AlertNotification.test.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Alert from './AlertNotification';
+import { Variant } from './Alerts.enum';
+import { CheckCircle, XCircle, Info, AlertTriangle } from 'lucide-react';
+
+const variantTestCases = [
+  {
+    variant: Variant.Success,
+    title: 'Success!',
+    message: 'This is a success message',
+    icon: CheckCircle,
+  },
+  {
+    variant: Variant.Error,
+    title: 'Error!',
+    message: 'This is an error message',
+    icon: XCircle,
+  },
+  {
+    variant: Variant.Default,
+    title: 'Info',
+    message: 'This is an info message',
+    icon: Info,
+  },
+  {
+    variant: Variant.Warning,
+    title: 'Warning!',
+    message: 'This is a warning message',
+    icon: AlertTriangle,
+  },
+];
+
+describe('AlertNotification', () => {
+  variantTestCases.forEach(({ variant, message }) => {
+    it(`renders the correct variant ${variant}`, () => {
+      render(<Alert variant={variant} message={message} />);
+
+      switch (variant) {
```

Comment:
This is incorrect. It should not be manually testing every individual variant. The data is already in the loop. The `alert-title` should equal `variant.title`. Same for the other variants.